app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingFragment.kt

```diff
@@ -12,15 +12,7 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.background
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxHeight
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.*
```

Comment:
Sorry, @indurs but now I noticed this * import. Can you please revert this and after that squash the commits? It's not a good practice to use * import. It's the last change I ask. Thanks!